src/SFA.DAS.AssessorService.Application.Api/Controllers/Apply/ApplicationController.cs

```diff
@@ -82,23 +82,13 @@ public async Task<ActionResult<ApplicationResponse>> GetApplication(string id)
             return Ok(await _mediator.Send(new GetApplicationRequest(Guid.Parse(id))));
         }
 
-
-        [HttpGet("{orgId}/application/withdrawn/{standardCode}", Name = "GetAllWithdrawnApplicationsForStandard")]
-        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(ApplicationResponse))]
-        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
-        public async Task<ActionResult<List<ApplicationResponse>>> GetAllWithdrawnApplicationsForStandard(string orgId, int? standardCode)
-        {
-            _logger.LogInformation($"Received request to retrieve withdrawn applications with OrganisationId {orgId}");
-            return Ok(await _mediator.Send(new GetAllWithdrawnApplicationsForStandardRequest(Guid.Parse(orgId), standardCode)));
-        }
-
-        [HttpGet("{orgId}/application/previous/{standardReference}", Name = "GetPreviousApplications")]
+        [HttpGet("{organisationId}/application/latest-withdrawal-date/{standardCode}", Name = "GetLatestWithdrawalDateForStandard")]
         [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(ApplicationResponse))]
         [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
-        public async Task<ActionResult<List<ApplicationResponse>>> GetApplicationsPrevious(Guid orgId, string standardReference)
+        public async Task<ActionResult<DateTime?>> GetLatestWithdrawalDateForStandard(string organisationId, int? standardCode)
         {
-            _logger.LogInformation($"Received request to retrieve previous applications with OrganisationId {orgId}");
-            return Ok(await _mediator.Send(new GetPreviousApplicationsRequest(orgId, standardReference)));
+            _logger.LogInformation($"Received request to retrieve withdrawal applications for Organisation {organisationId} with standard {standardCode}");
+            return Ok(await _mediator.Send(new GetLatestWithdrawalDateForStandardRequest(Guid.Parse(organisationId), standardCode)));
         }
```

Comment:
Log injection vulnerability here too https://sonarcloud.io/project/issues?resolved=false&types=VULNERABILITY&pullRequest=1557&id=SkillsFundingAgency_das-assessor-service&open=AYohiQjGao39yYF9-WnO

Reply:
I have resolved that as won't fix as we do not share the logs with other applications